docs/source/configuration.md

```diff
@@ -1,24 +1,24 @@
 # Configuration
```

Comment:
I removed the `--` preceding each parameter becasue that's usually for CLI parameters.